### DISCUSSION TITLE
No ambient lighting from the screens

### DISCUSSION THREAD

**Discussion by FoodAviation** [flybywiresim/aircraft#2909] [Help] [answered]
Hi! Shortly, I don't have ambient light from the screens like here:
![esnqnxn9h0p51](https://user-images.githubusercontent.com/77068219/103796219-3b234780-5047-11eb-8b71-20db6f92e2e4.jpg)

My cockpit looks like this: 
![Zrzut ekranu (119)](https://user-images.githubusercontent.com/77068219/103796116-1dee7900-5047-11eb-8180-9ee3057f9b14.png)

And like this with flood light on:
![Zrzut ekranu (120)](https://user-images.githubusercontent.com/77068219/103797840-44adaf00-5049-11eb-9b0b-65882dcf684b.png)

I have the latest, stable version 0.5.1. I tried it on default liveries. Except of that everything work ok. I have GTX1060 and graphical settings like high and ultra. I tried also to reinstall it but didn't help.

What can be wrong? Please help.

**Comment by MapleNAV-TCASFAULT** [comment]


**Reply by pepperoni505** [reply]
The poster is referring to the display backlights

**Comment by pepperoni505** [comment, marked as answer]
In 0.5.0, there was a lighting overhaul that removed the display backlight due to the fact that it wasn't realistic. When it was added we were under the impression it was due to photos, but since then IRL pilots have confirmed it shouldn't be that way
